.github/workflows/push-frontend.yml

```diff
@@ -0,0 +1,51 @@
+name: Build NodeJS Image and Push to Openshift Registry
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+    branches: [main]
+
+jobs:
+  build-main:
+    runs-on: ubuntu-20.04
+    env:
+      VERSION: v1
+      IMAGE_NAME: citizen-portal
+      IMAGE_TAGS: latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # - name: Ensure actor is clean
+      #   id: clean-actor
+      #   env:
+      #     COMPOSE_DOCKER_CLI_BUILD: 1
+      #     DOCKER_BUILDKIT: 1
+      #   run: |          
+      #     docker system prune -a --force
+
+      - name: Build the Image
+        id: build-image
+        env:
+          COMPOSE_DOCKER_CLI_BUILD: 1
+          DOCKER_BUILDKIT: 1
+        run: |          
+          docker login ${{ secrets.REDHAT_REPO }} -u ${{ secrets.REDHAT_REPO_USER }} -p ${{ secrets.REDHAT_REPO_AUTH }}
```

Comment:
Separate step for login ? looks like login should go with push or standalone step ?

Reply:
oh just seing this, that's the redhat repo

Reply:
are you using redhat images ?

Reply:
Yes, brought this up in standup.  We had issues getting some containers to stand up in OpenShift because they didn't meet security rules.  I ran into this before on the Prime project and burned about 40 hours trying to work around - it's not worth it. Phil and Ranti agreed, we should ensure devs have a RedHat developer account.